source/developer/slash-commands.rst

```diff
@@ -53,6 +53,12 @@ Each Mattermost installation comes with some built-in slash commands that are re
     "/settings", "Open the Account Settings dialog", "/settings"
     "/shortcuts", "Display a list of keyboard shortcuts", "/shortcuts"
     "/shrug *{message}*", "Add ``¯\_(ツ)_/¯`` to your message", "/shrug oh well"
+    "/status *{emoji_name}* *{descriptive status_message}* *", Set a custom status that includes an optional emoji and a descriptive status message, "/status sick Feeling unwell and taking time off to recover"
+    "/status clear *", Clear the current status, "/status clear"
+
+.. note::
+
+  Custom status slash commands marked with an asterisk (*) in the table above will be available in the Mattermost Mobile App in a future release.
```

Comment:
```suggestion
  Custom status slash commands marked with an asterisk [*] in the table above will be available in the Mattermost Mobile App in a future release.
```

Reply:
Wondering if this fixes the problem (the following text is italicized) or makes it worse. Not sure how we handle the addition of the * in other docs, will have to look around.

Reply:
@justinegeffen Given the formatting complexity of including asterisks, I'm going to transition to another approach altogether. Instead of marking specific Slash Commands with asterisks, I'm simply listing the 2 commands coming to mobile soon.